acme/setup.py

```diff
@@ -22,15 +22,14 @@
 ]
 
 # env markers in extras_require cause problems with older pip: #517
+# Keep in sync with conditional_requirements.py.
 if sys.version_info < (2, 7):
     install_requires.extend([
         # only some distros recognize stdlib argparse as already satisfying
         'argparse',
-        'mock<1.1.0',
```

Comment:
Would you mind actually addressing my concerns and suggested solution instead of ignoring it? I don't see any relevant discussion in IRC either.

To make it explicit: please don't remove `mock` dependency (at least as a part of this PR). Thanks!